DPC++Compiler/vector-add/src/vector-add.cpp

```diff
@@ -111,7 +111,7 @@ void VectorAddInDPCPP(const IntArray &addend_1, const IntArray &addend_2,
   // contains the data access permission and device computation (kernel)
   q.submit([&](handler &h) {
     // create an accessor for each buffer with access permission: read, write or
-    // read/write the accessor is the only mean to access the memory in the
+    // read/write. The accessor is the only means to access the memory in the

```

Comment:
Curious to have plural "means" here.

Reply:
[here is the definition](https://www.collinsdictionary.com/us/dictionary/english/means)